<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1044</width>
    <height>747</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Pep/8</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout">
    <item>
     <widget class="QSplitter" name="horizontalSplitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QSplitter" name="leftSplitter">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QTabWidget" name="pepStackTraceTab">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="pepCodeTab">
         <attribute name="title">
          <string>Code</string>
         </attribute>
         <layout class="QVBoxLayout">
          <item>
           <widget class="QSplitter" name="codeSplitter">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="handleWidth">
             <number>7</number>
            </property>
            <widget class="QWidget" name="SourceCodeWidgetPane" native="true"/>
            <widget class="QWidget" name="ObjectCodeWidgetPane" native="true"/>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="pepTraceTab">
         <attribute name="title">
          <string>Trace</string>
         </attribute>
         <layout class="QVBoxLayout">
          <item>
           <widget class="QSplitter" name="traceSplitter">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <widget class="QWidget" name="MemoryTraceWidgetPane" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="AssemblerListingWidgetPane" native="true"/>
      </widget>
      <widget class="QSplitter" name="middleSplitter">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="handleWidth">
        <number>7</number>
       </property>
       <widget class="QWidget" name="CpuWidgetPane" native="true"/>
       <widget class="QTabWidget" name="pepInputOutputTab">
        <property name="tabShape">
         <enum>QTabWidget::Rounded</enum>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="pepBatchTab">
         <attribute name="title">
          <string>Batch I/O</string>
         </attribute>
         <layout class="QVBoxLayout">
          <item>
           <widget class="QSplitter" name="batchSplitter">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="handleWidth">
             <number>7</number>
            </property>
            <widget class="QWidget" name="InputWidgetPane" native="true">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
            <widget class="QWidget" name="OutputWidgetPane" native="true"/>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="pepTerminalTab">
         <attribute name="title">
          <string>Terminal I/O</string>
         </attribute>
         <layout class="QVBoxLayout">
          <item>
           <widget class="QSplitter" name="terminalSplitter">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <widget class="QWidget" name="TerminalWidgetPane" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="MemoryDumpWidgetPane" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="fileToolBar">
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionPrint"/>
  </widget>
  <widget class="QToolBar" name="editToolBar">
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="actionCut"/>
   <addaction name="actionCopy"/>
   <addaction name="actionPaste"/>
  </widget>
  <widget class="QToolBar" name="buildToolBar">
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAssemble"/>
   <addaction name="actionLoad"/>
   <addaction name="actionExecute"/>
  </widget>
  <widget class="QToolBar" name="viewToolBar">
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionCode_Only"/>
   <addaction name="actionCode_CPU"/>
   <addaction name="actionCode_CPU_Memory"/>
   <addaction name="actionCode_Memory"/>
  </widget>
  <widget class="QToolBar" name="converterToolBar">
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1044</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionFont"/>
    <addaction name="separator"/>
    <addaction name="actionFind"/>
   </widget>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionPrint"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuBuild">
    <property name="title">
     <string>Build</string>
    </property>
    <addaction name="actionAssemble"/>
    <addaction name="actionLoad"/>
    <addaction name="actionExecute"/>
    <addaction name="separator"/>
    <addaction name="actionAssemble_Load"/>
    <addaction name="actionLoad_Execute"/>
    <addaction name="actionAssemble_Load_Execute"/>
    <addaction name="separator"/>
    <addaction name="actionRemove_Error_Messages"/>
   </widget>
   <widget class="QMenu" name="menu_System">
    <property name="title">
     <string>&amp;System</string>
    </property>
    <addaction name="actionRedefine_Mnemonics"/>
    <addaction name="separator"/>
    <addaction name="actionAssemble_Install_New_OS"/>
    <addaction name="actionReinstall_Default_OS"/>
    <addaction name="separator"/>
    <addaction name="actionSet_Execution_Limits"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionCode_Only"/>
    <addaction name="actionCode_CPU"/>
    <addaction name="actionCode_CPU_Memory"/>
    <addaction name="actionCode_Memory"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <widget class="QMenu" name="menuPep_8_Reference">
     <property name="title">
      <string>Pep/8 Reference</string>
     </property>
     <addaction name="actionHelp_Instruction_Set"/>
     <addaction name="actionHelp_Dot_Commands"/>
     <addaction name="actionHelp_Addressing_Modes"/>
    </widget>
    <widget class="QMenu" name="menuWriting_Programs">
     <property name="title">
      <string>Writing Programs</string>
     </property>
     <addaction name="actionHelp_Assembly_Language"/>
     <addaction name="actionHelp_Machine_Language"/>
    </widget>
    <widget class="QMenu" name="menuDebugging_programs">
     <property name="title">
      <string>Debugging Programs</string>
     </property>
     <addaction name="actionHelp_Break_Points"/>
     <addaction name="actionHelp_Symbolic_Trace"/>
     <addaction name="actionHelp_Byte_Converter"/>
    </widget>
    <addaction name="menuWriting_Programs"/>
    <addaction name="menuDebugging_programs"/>
    <addaction name="actionHelp_Writing_Interrupt_Handlers"/>
    <addaction name="menuPep_8_Reference"/>
    <addaction name="separator"/>
    <addaction name="actionAbout_Pep8"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_View"/>
   <addaction name="menuBuild"/>
   <addaction name="menu_System"/>
   <addaction name="menu_Help"/>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="pep8resources.qrc">
     <normaloff>:/images/new.png</normaloff>:/images/new.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="pep8resources.qrc">
     <normaloff>:/images/open.png</normaloff>:/images/open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="pep8resources.qrc">
     <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset resource="pep8resources.qrc">
     <normaloff>:/images/print.png</normaloff>:/images/print.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Print...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="pep8resources.qrc">
     <normaloff>:/images/undo.png</normaloff>:/images/undo.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="pep8resources.qrc">
     <normaloff>:/images/redo.png</normaloff>:/images/redo.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="pep8resources.qrc">
     <normaloff>:/images/cut.png</normaloff>:/images/cut.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="pep8resources.qrc">
     <normaloff>:/images/copy.png</normaloff>:/images/copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="pep8resources.qrc">
     <normaloff>:/images/paste.png</normaloff>:/images/paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionFont">
   <property name="text">
    <string>Font...</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
  </action>
  <action name="actionAssemble">
   <property name="icon">
    <iconset resource="pep8resources.qrc">
     <normaloff>:/images/assemble.png</normaloff>:/images/assemble.png</iconset>
   </property>
   <property name="text">
    <string>Assemble</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+A</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="icon">
    <iconset resource="pep8resources.qrc">
     <normaloff>:/images/load.xpm</normaloff>:/images/load.xpm</iconset>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+L</string>
   </property>
  </action>
  <action name="actionExecute">
   <property name="icon">
    <iconset resource="pep8resources.qrc">
     <normaloff>:/images/execute.png</normaloff>:/images/execute.png</iconset>
   </property>
   <property name="text">
    <string>Execute</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+E</string>
   </property>
  </action>
  <action name="actionAssemble_Load">
   <property name="text">
    <string>Assemble/Load</string>
   </property>
  </action>
  <action name="actionLoad_Execute">
   <property name="text">
    <string>Load/Execute</string>
   </property>
  </action>
  <action name="actionAssemble_Load_Execute">
   <property name="text">
    <string>Assemble/Load/Execute</string>
   </property>
  </action>
  <action name="actionRemove_Error_Messages">
   <property name="text">
    <string>Remove Error Messages</string>
   </property>
  </action>
  <action name="actionRedefine_Mnemonics">
   <property name="text">
    <string>Redefine Mnemonics...</string>
   </property>
  </action>
  <action name="actionAssemble_Install_New_OS">
   <property name="text">
    <string>Assemble/Install New OS</string>
   </property>
  </action>
  <action name="actionReinstall_Default_OS">
   <property name="text">
    <string>Reinstall Default OS</string>
   </property>
  </action>
  <action name="actionSet_Execution_Limits">
   <property name="text">
    <string>Set Execution Limits...</string>
   </property>
  </action>
  <action name="actionHelp_Writing_Interrupt_Handlers">
   <property name="text">
    <string>Writing Interrupt Handlers</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="text">
    <string>Find...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionCode_Only">
   <property name="icon">
    <iconset resource="pep8resources.qrc">
     <normaloff>:/images/CodeOnly.xpm</normaloff>:/images/CodeOnly.xpm</iconset>
   </property>
   <property name="text">
    <string>Code Only</string>
   </property>
  </action>
  <action name="actionCode_CPU">
   <property name="icon">
    <iconset resource="pep8resources.qrc">
     <normaloff>:/images/CodeCPU.xpm</normaloff>:/images/CodeCPU.xpm</iconset>
   </property>
   <property name="text">
    <string>Code/CPU</string>
   </property>
  </action>
  <action name="actionCode_CPU_Memory">
   <property name="icon">
    <iconset resource="pep8resources.qrc">
     <normaloff>:/images/CodeCPUMemory.xpm</normaloff>:/images/CodeCPUMemory.xpm</iconset>
   </property>
   <property name="text">
    <string>Code/CPU/Memory</string>
   </property>
  </action>
  <action name="actionCode_Memory">
   <property name="icon">
    <iconset resource="pep8resources.qrc">
     <normaloff>:/images/CodeMemory.xpm</normaloff>:/images/CodeMemory.xpm</iconset>
   </property>
   <property name="text">
    <string>Code/Memory</string>
   </property>
  </action>
  <action name="actionAbout_Pep8">
   <property name="text">
    <string>About Pep8</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About QT</string>
   </property>
  </action>
  <action name="actionHelp_Assembly_Language">
   <property name="text">
    <string>Assembly Language</string>
   </property>
  </action>
  <action name="actionHelp_Machine_Language">
   <property name="text">
    <string>Machine Language</string>
   </property>
  </action>
  <action name="actionHelp_Break_Points">
   <property name="text">
    <string>Break Points</string>
   </property>
  </action>
  <action name="actionHelp_Symbolic_Trace">
   <property name="text">
    <string>Symbolic Trace</string>
   </property>
  </action>
  <action name="actionHelp_Instruction_Set">
   <property name="text">
    <string>Instruction Set</string>
   </property>
  </action>
  <action name="actionHelp_Dot_Commands">
   <property name="text">
    <string>Dot Commands</string>
   </property>
  </action>
  <action name="actionHelp_Byte_Converter">
   <property name="text">
    <string>Byte Converter</string>
   </property>
  </action>
  <action name="actionHelp_Addressing_Modes">
   <property name="text">
    <string>Addressing Modes</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="pep8resources.qrc"/>
 </resources>
 <connections/>
</ui>
